Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity

```diff
@@ -14675,7 +14501,7 @@ PrefabInstance:
     - target: {fileID: 4943773361295851263, guid: c0931c4da6d91ea429abedb10290dd16,
         type: 3}
       propertyPath: m_IsActive
-      value: 1
+      value: 0
```

Comment:
This change looks like it turned off the features panel by default :( should this be reverted as well? @cre8ivepark @davidkline-ms